release-calendar/src/entities/release.ts

```diff
@@ -14,17 +14,33 @@
  * limitations under the License.
  */
 
-import {Entity, PrimaryColumn} from 'typeorm';
+import {Entity, PrimaryColumn, Column} from 'typeorm';
+import {Channel} from '../types';
 
 @Entity()
 export class Release {
-
-  constructor(name: string) {
+  constructor(name: string, channel: Channel, isRollback: boolean, date: Date) {
```

Comment:
Since `isRollback` is such a rare occurrence, how do you feel about defaulting it to false and setting it as an optional parameter in the constructor?